src/Plugins/DavisIMipPlugin.php

```diff
@@ -75,6 +75,15 @@ public function schedule(ITip\Message $itip)
         $senderEmail = substr($itip->sender, 7);
         $recipientEmail = substr($itip->recipient, 7);
 
+        if ($itip->senderName) {
+            $senderName = $itip->senderName.' <'.$senderEmail.'>';
```

Comment:
This is probably not wanted, as it will create the following header in the email:
```
From: "NAME <email@email.com> (via Davis)" <email@email.com>
```

You want to leave the display of the tuple (name, email) to the mail client

Reply:
As it is this will give:
If senderName is present: `From: "NAME <email@email.com> (via Davis)" <email@email.com>` 
If senderName is not present: `From: "email@email.com (via Davis)" <email@email.com>`

The alternative is if senderName is not present: `From: "(via Davis)" <email@email.com>`

This fix would only apply to clients that aren't setting senderName in the iMip as referenced in issue #165

Reply:
I see what you mean, this can be amended to just set senderName to the iTip senderName in that case.

Reply:
In this case, the (imo) correct behavior si to _not_ add `MESSAGE_ORIGIN_INDICATOR` if `$senderName` is empty

Reply:
I think I'll find a middle ground, give me a minute

Reply:
This should be a good compromise: https://github.com/tchapi/davis/commit/231b7ffb3dfeeac42ddeb72c10e60c61924db4c5

Tell me what you think

Reply:
Looks good, that will work for me. Thank you!